src/inputs/button/button.jsx

```diff
@@ -313,12 +316,14 @@ class Button extends React.PureComponent {
             id,
             innerStyle,
             inverse,
-            outlined,
+            outline,
             relax,
             style,
             target,
             title,
             width,
+            pill,
+            transparent
         } = this.props;
         const disable = disableProp || disabled;
```

Comment:
Please remove `disabled` for all version 10.0.0 gap closures. It was a prop that was deprecated in version 9.